grpchantesting/channel_test_cases.go

```diff
@@ -8,14 +8,13 @@ import (
 	"testing"
 	"time"
 
-	"github.com/golang/protobuf/proto"
-	"github.com/golang/protobuf/ptypes"
-	"github.com/golang/protobuf/ptypes/any"
-	"github.com/golang/protobuf/ptypes/struct"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/metadata"
 	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/anypb"
+	"google.golang.org/protobuf/types/known/structpb"
```

Comment:
^^ is this part of the 1.3 -> 1.4 migration, generally?

Reply:
Kind of. I mentioned this in my PR description: the old packages are now deprecated. So I went ahead and did this to make `staticcheck` not complain about usage of deprecated things. (Seemed simple enough -- and better than either ignoring that check in the Makefile or littering all of these files with "linter:ignore" directives.)